src/index.js

```diff
@@ -0,0 +1,30 @@
+let input = document.getElementById('operationInput');
```

Comment:
The `input` variable it's a `const`, you are only changing the properties

Reply:
aight